app/controllers/comments_controller.rb

```diff
@@ -6,7 +6,13 @@ def new
   def create
     @post = Post.find(params[:post_id])
     @comment = @post.comments.create(comment_params)
-    redirect_to user_post_path(User.find(params[:user_id]), @post)
+    respond_to do |format|
+      if @post.save
+        format.html do
+          redirect_to user_post_path(User.find(params[:user_id]), @post), notice: 'Comment successfully created!'
+        end
+      end
```

Comment:
- [ ] Let's also add a flash message for a not successful path. Kindly add a flash message for when the comment is not successfully created.